### PR TITLE
adding docs on flow.visualuze()

### PR DESCRIPTION
**Pull request by WillRaphaelson**
adding docs on flow.visualuze()

### Checklist

- [x] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`, `docs`.
  

For documentation changes:

- [x] This pull request includes redirect settings in `netlify.toml` for files that are removed or renamed

### CODE REVIEW

File: docs/concepts/flows.md [hunk 1]
```diff
@@ -251,6 +251,61 @@ Hello Marvin!
...
+```
+
+image
+
+In some cases, such as dynamic workflows with loops or if/else control flow, Prefect is unable to automatically produce a schematic. In this case, you can provide tasks with mock return values for use in the `visualize()` call.
+
+```python
+from prefect import task, flow
+
+@task(vis_return_value=["123a","456b", "789c"]) # necessary to produce a schematic without actually running the flow code
```
Comment (serinamarie) [inline docs/concepts/flows.md:288]:
```suggestion
@task(viz_return_value=["123a","456b", "789c"]) # necessary to produce a schematic without actually running the flow code
```

File: docs/concepts/flows.md [hunk 2]
```diff
@@ -251,6 +251,61 @@ Hello Marvin!
 ```
 </div>
 
+## Visualizing Flow Structure
```
Comment (serinamarie) [inline docs/concepts/flows.md:254]:
```suggestion
## Visualizing Flow Structure <span class="badge experimental"></span>
```

File: docs/concepts/flows.md [hunk 3]
```diff
@@ -251,6 +251,61 @@ Hello Marvin!
...
+def hello_world(name="world"):
+    message = print_hello(name)
+    message2 = print_hello_again(message)
+
+hello_world.visualize()
+```
+
+![A simple flow visualized with the .visualize() method](/img/orchestration/hello-flow-viz.png)
+
+In some cases, such as dynamic workflows with loops or if/else control flow, Prefect is unable to automatically produce a schematic. In this case, you can provide tasks with mock return values for use in the `visualize()` call.
```
Comment (billpalombi) [inline docs/concepts/flows.md:285]:
```suggestion
Prefect cannot automatically produce a schematic for dynamic workflows, such as those with loops or if/else control flow. In this case, you can provide tasks with mock return values for use in the `visualize()` call.
```

File: docs/concepts/flows.md [hunk 4]
```diff
@@ -251,6 +251,61 @@ Hello Marvin!
 ```
 </div>
 
+## Visualizing Flow Structure <span class="badge experimental"></span>
+
+You can get a quick sense of the structure of your flow using the `.visualize()` method on your flow. Calling this method will attempt to produce a schematic diagram of your flow and tasks without actually running your flow code.
```
Comment (billpalombi) [inline docs/concepts/flows.md:256]:
```suggestion
You can get a quick sense of the structure of your flow using the `.visualize()` method. Calling this method will attempt to produce a schematic diagram of your flow and tasks without actually running your flow code.
```

**Review by billpalombi — APPROVED** [review]
A couple of nits. Otherwise lgtm

File: docs/concepts/flows.md [hunk 5]
```diff
@@ -251,6 +251,61 @@ Hello Marvin!
...
+
```
Comment (serinamarie) [inline docs/concepts/flows.md:257]: note that it does run your code if it's not in a task or flow

**Pull request merged**